base/strings/util.jl

```diff
@@ -248,6 +248,123 @@ julia> rpad("March",20)
 rpad(s, n::Integer, p=" ") = rpad(string(s),n,string(p))
 cpad(s, n::Integer, p=" ") = rpad(lpad(s,div(n+strwidth(s),2),p),n,p)
 
+immutable SplitIterator
+    str::AbstractString
+    splitter
```

Comment:
I believe regular `split(str, "")` will split graphemes up. That might or might not be a good thing; I think it's not ideal.